scripts/pre_commit_linter.py

```diff
@@ -1018,41 +1021,51 @@ def _check_docstrings(all_files):
                 if line_num > 0:
                     prev_line = file_content[line_num - 1].lstrip().rstrip()
 
+                if (prev_line.startswith('class ') or
```

Comment:
Sometimes "class" and "def" definitions span more than one line, like [this one](https://github.com/oppia/oppia/blob/develop/core/storage/base_model/gae_models.py#L197). How do you account for that?

Reply:
Done